### ISSUE TITLE
Migrations ignored from git_commit_id when the app label does not exactly match the folder name

### ISSUE THREAD

**Issue by phillipuniverse** [3YOURMIND/django-migration-linter#249]
Here's a screenshot from the Pycharm debugger that shows the issue

<img width="1861" alt="image" src="https://user-images.githubusercontent.com/684275/216882535-33cb19c9-0c57-47ab-b45e-3ef4fcefe6c7.png">

The migration linter is checking for a migration with a tuple `("netsuite", "0012_added_netsuite_suiteapp_config")` against the Django `disk_migrations` (`self.migration_loader` is the Django `MigrationLoader`).

This migration linter assumes that "netsuite" is a valid app name for Django. But it's not - Django doesn't know about an app called `netsuite`, it knows about one called `integrations_netsuite`. This is because we renamed it in a custom `AppConfig` in `apps.py`:

```python
 class NetsuiteConfig(AppConfig): 
     name = "integrations.netsuite" 
     label = name.replace(".", "_") 
```

So the root issue is that the linter doesn't look truly at app labels, it only looks at folder structures.

This appears to only be an issue when used in concert with the git commit id.

Fix for this issue is at #248 which takes the folder structure and converts it to the real Django app without assumptions about naming.

**Comment by David-Wobrock** [comment]
Hi @phillipuniverse 

Thanks for the report! That's indeed something the linter should support.
I'll try to look into it and review https://github.com/3YOURMIND/django-migration-linter/pull/248 soon :)

**Comment by mjvankampen** [comment]
Ran into this as well!
